website/settings/defaults.py

```diff
@@ -2067,3 +2074,7 @@ def from_node_usage(cls,  usage_bytes, private_limit=None, public_limit=None):
 ID_VERSION = 'staging_v2'
 IA_ROOT_COLLECTION = 'cos-dev-sandbox'
 PIGEON_CALLBACK_BEARER_TOKEN = os.getenv('PIGEON_CALLBACK_BEARER_TOKEN')
+
+PRODUCT_OWNER_EMAIL_ADDRESS = {
+    'Registration': 'mark@cos.io',
```

Comment:
As a general rule, we don't want to store real emails in the repo, because other people *occasionally* run copies of the OSF and we don't want to email real email addresses on their local installations.

Reply:
Yep, I will change this to use system envs.

Reply:
In addition, this also reminds to see if we can set the schedule via env so that prod has a lower frequency (every 1-3 hours) while others have a higher one (5 minutes).